features/upgrade/api_auth/upgrade.feature

```diff
@@ -328,4 +328,4 @@ Feature: apiserver and auth related upgrade check
       | n        | ocp-34223-proj |
     Then the step should succeed
     # This is to discover bugs like: 1845411 1804717 1912820
-    And the output should not contain "failed"
+    And the expression should be true> @result[:response].scan(/failed/).length <= 2
```

Comment:
The PR is for fix for one time failed error but upgrade success. Why two 'failed' can be tolerated in log file?  In jira issue OCPQE-5300, we got the failed message in case log, need specific analysis, for failed cases, it's better to run must-gather, check detail with that.

Reply:
@wangke19 sometimes we got error failed but there is no issue in upgrade and upgrade is success like this http://10.14.89.4:3000/url/generate?key=logs/2021/12/14/05:51:52/kube-apiserver_and_openshift-apiserver_should_have_zero-disruption_upgrade. Sometimes it can be 2 "failed" messages. that's why I ignore 2 failed messages.

Reply:
Better to tolerate only length <= 1

Reply:
Updated length to 1